packages/contentstack-clone/src/commands/cm/stacks/clone.js

```diff
@@ -23,7 +23,7 @@ class StackCloneCommand extends Command {
         'source-management-token-alias': sourceManagementTokenAlias,
         'destination-management-token-alias': destinationManagementTokenAlias,
         'import-webhook-status': importWebhookStatus,
-        'master-locale': masterLocale
+        // 'master-locale': masterLocale
```

Comment:
Remove commented code

Reply:
Netraj, I had commented the code, in case someone raises an objection to removing this flag and we have to reintroduce it.